metadata-service/configuration/src/main/resources/application.yaml

```diff
@@ -370,7 +370,12 @@ systemUpdate:
     batchSize: ${SYSTEM_UPDATE_SCHEMA_FIELDS_DOC_IDS_BATCH_SIZE:500}
     delayMs: ${SYSTEM_UPDATE_SCHEMA_FIELDS_DOC_IDS_DELAY_MS:5000}
     limit: ${SYSTEM_UPDATE_SCHEMA_FIELDS_DOC_IDS_LIMIT:0}
-
+  processInstanceHasRunEvents:
```

Comment:
Missing `delayMs` maybe the code isn't pushed?

Reply:
Yes, apologies - pushed now.